pygmo/core.cpp

```diff
@@ -646,7 +651,8 @@ BOOST_PYTHON_MODULE(core)
     pygmo::expose_problem<tu_test_problem>("_tu_test_problem", "A thread unsafe test problem.");
     // Null problem.
     auto np = pygmo::expose_problem<null_problem>("null_problem", pygmo::null_problem_docstring().c_str());
-    np.def(bp::init<vector_double::size_type>((bp::arg("nobj"))));
+    np.def(bp::init<vector_double::size_type, vector_double::size_type, vector_double::size_type>(
```

Comment:
Is this gonna work if one specifies only ``nec`` in the constructor on the Python side? I feel like ``nobj`` should have a default value as well, but then I don't know if it clashes with the def ctor already exposed by ``expose_problem``.